test/quantization/test_quantize_script.py

```diff
@@ -50,9 +50,9 @@ class TestQuantizeScriptJitPasses(QuantizationTestCase):
     """
     def test_foldbn_trivial(self):
         # Test trivial case
-        class TestModule(torch.nn.Module):
+        class TestModule2d(torch.nn.Module):
```

Comment:
can you follow: https://github.com/pytorch/pytorch/blob/master/test/quantization/test_quantize_script.py#L1771 and merge the two class into one?